### ISSUE TITLE
Issue when using `netlify_team_firewall_traffic_rules` resource

### ISSUE THREAD

**Issue by janario** [netlify/terraform-provider-netlify#41]
We are applying some `netlify_team_firewall_traffic_rules` to make sure apps are kept under our ip ranges.

```
data "netlify_team" "my_team" {
  slug = var.team_slug
}

locals {
  ip_exceptions = [
    {
      description = "internal-ips"
      addresses = [
        "<our ip range>"
      ]
     }
  ]
}

resource "netlify_team_firewall_traffic_rules" "team_rules" {
  team_id = data.netlify_team.my_team.id
  published = {
    default_action = "deny"
    ip_exceptions = local.ip_exceptions
  }
  unpublished = {
    default_action = "deny"
    ip_exceptions = local.ip_exceptions
  }
}
```

The first initial apply works fine but after a re-plan it fails on reading it.

```

Planning failed. Terraform encountered an error while generating this plan.

╷
│ Error: Error reading team firewall rule set
│
│   with netlify_team_firewall_traffic_rules.team_rules,
│   on main.tf line 86, in resource "netlify_team_firewall_traffic_rules" "team_rules":
│   86: resource "netlify_team_firewall_traffic_rules" "team_rules" {
│
│ Could not read team firewall rule set "<team id>": "id is required and must be specified"
╵
```

It looks like somewhere here we would need to store the rule id in the state file and re-use it when reading https://github.com/netlify/terraform-provider-netlify/blob/main/internal/provider/firewall_traffic_rules_resource.go#L43

**Comment by ramonsnir** [comment]
Thank you for reporting this, @janario!

One of our goals for the v1.0.0 release is full automated testing for all resources, but as you can see this resource wasn't stable enough.

The fix will be included in v0.0.4 early next week, Monday or Tuesday.

**Comment by janario** [comment]
Awesome, thanks for the quick fix
